monitor/internal/docker/monitor.go

```diff
@@ -654,14 +657,27 @@ func (d *DockerMonitor) setupDockerDaemon() (err error) {
 func (d *DockerMonitor) waitForDockerDaemon(ctx context.Context) (err error) {
 
 	done := make(chan bool)
-	go func() {
-		for errg := d.setupDockerDaemon(); errg != nil; {
+	go func(gctx context.Context) {
```

Comment:
For user experience, let's add an Info message here: "Trying to initialize docker monitor ... "

Reply:
Done